patroni/postgresql/postmaster.py

```diff
@@ -123,7 +124,7 @@ def signal_stop(self, mode):
             elif not self.is_running():
                 return True
             else:
-                logger.warning("Could not send stop signal %s to postmaster with PID %s", STOP_SIGNALS[mode], self.pid)
+                logger.warning("Could not send stop signal to PostgreSQL with PID {0}".format(self.pid))
```

Comment:
Why this change? `%s` formatting is preferred with the logging module

Reply:
To be consistent with the similar log message several lines before :)

Reply:
I would actually like to have the other line fixed too, for consistency :)